### PR TITLE
docs: update docs for upcoming Sidero 0.4.1 release

### PR DESCRIPTION
**Pull request by smira**
See https://github.com/talos-systems/sidero/pull/669

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

### CODE REVIEW

File: website/content/docs/v0.4/Guides/first-cluster.md [hunk 1]
```diff
@@ -28,8 +28,16 @@ kubectl patch deploy -n sidero-system sidero-controller-manager --type='json' -p
 
 #### Update Environment
 
-The metadata server's information needs to be updated in the default environment.
-Edit the environment with `kubectl edit environment default` and update the `talos.config` kernel arg with the IP of one of the management plane nodes (or the DNS entry you created).
+The metadata server's information might need to be updated in the default environment.
```
Comment (frezbo) [inline website/content/docs/v0.4/Guides/first-cluster.md:31]: is this the metadata server or the sidero controller?

**Review by frezbo — CHANGES_REQUESTED** [review]

**Review by Unix4ever — APPROVED** [review]

File: website/content/docs/v0.4/Guides/first-cluster.md (hunk 1 above)
Comment (smira) [reply website/content/docs/v0.4/Guides/first-cluster.md:31]: Hmm, looks like I can probably remove whole section there

**Review by frezbo — APPROVED** [review]

**Pull request merged**